RestApiApp/DateExtension.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  DateExtension.swift
+//  RestApiApp
+//
+//  Created by Nele Müller on 25.05.18.
+//  Copyright © 2018 Nele Müller. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
+    static func fromString(date: String) -> Date {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+        
+        guard let returnDate = dateFormatter.date(from: date) else {
+            fatalError("Date couldn't be created from String")
```

Comment:
Tip: Try to avoid `fatalError()` calls in production code. For developing and testing maybe it's ok, but I think noone want a `not recoverable` error.